### DISCUSSION TITLE
Starting the dataflow locally

### DISCUSSION THREAD

**Discussion by Isuc0100** [bytewax/bytewax#257] [Q&A] [answered]
I had a question regarding executing the dataflow locally, like for example a Kafka streams app what method would I utilize in order to start the dataflow after creating the logic

**Comment by davidselassie** [comment, marked as answer]
This is described in the [execution](https://bytewax.io/docs/getting-started/execution) section of our [long-form docs](https://bytewax.io/docs) on the website.

tl;dr: Store your configured `Dataflow` into a global variable named `flow` and then use `python -m bytewax.run $MODULE`. Note that the module name is an import path, not a filename.
